README.md

```diff
@@ -10,6 +10,40 @@ This repository hosts scripts to generate self-contained installations of the [E
 
 ------
 
+
+### 1.1. Prerequisites
+
+If you don't have Git, install it. You can install Git from websites such as [Git Downloads](https://git-scm.com/downloads).
+
+The bundled installer requires the following prerequisites for each operating system.
+
+- **Linux**
+    - **Ubuntu and Debian**
+
+        ```shell
+        build-essential zlib1g-dev libssl-dev libncurses-dev libffi-dev libsqlite3-dev libreadline-dev libbz2-dev
+        ```
+
+    - **Amazon Linux and Fedora**
+
+       ```shell
+       "Development Tools" zlib-devel openssl-devel ncurses-devel libffi-devel sqlite-devel.x86_64 readline-devel.x86_64 bzip2-devel.x86_64
+       ```
+
+- **macOS**
+
+     ```shell
+     openssl zlib readline
+     ```
+
+- **Windows**
+
+    ```shell
+    virtualenv
```

Comment:
This is meaningless. To install virtualenv, you need Python and pip.

Reply:
This was a proposed work around for the issue seen here https://github.com/aws/aws-elastic-beanstalk-cli-setup/issues/19. Although you're right, they only ran into that issue after having installed pip and python. So perhaps the troublehsooting section on this topic is enough for now.

Reply:
Yes.